core/server/src/main/java/alluxio/master/file/FileSystemMaster.java

```diff
@@ -510,15 +512,14 @@ public PersistenceState getPersistenceState(long fileId) throws FileDoesNotExist
    * {@link FileSystemAction#EXECUTE} permission on the path if it is a directory.
    *
    * @param path the path to get the {@link FileInfo} list for
-   * @param loadDirectChildren whether to load the direct children if path is a directory if its
-   *        direct children have not beed loaded before
+   * @param getFileInfoListOptions the {@link GetFileInfoListOptions}
    * @return the list of {@link FileInfo}s
    * @throws AccessControlException if permission checking fails
    * @throws FileDoesNotExistException if the file does not exist
    * @throws InvalidPathException if the path is invalid
    */
-  // TODO(peis): Create GetFileInfoListOptions and add an option not to load metadata.
-  public List<FileInfo> getFileInfoList(AlluxioURI path, boolean loadDirectChildren)
+  public List<FileInfo> getFileInfoList(AlluxioURI path,
```

Comment:
while we are doing this, can we rename this function to `listStatus` and its options to ListStatusOptions? thanks